src/toast/tests/_helpers.py

```diff
@@ -611,7 +956,153 @@ def create_fake_sky_tod(
         ],
     )
     scan_pipe.apply(data)
-    return map_key
+
+    # Cleanup, to avoid any conflicts with the calling code.
+    for ob in data.obs:
+        for buf in [
+            pixel_pointing.pixels,
+            stokes_weights.weights,
+            pixel_pointing.detector_pointing.quats,
+        ]:
+            if buf in ob:
+                del ob[buf]
+
+
+# def create_fake_sky(data, dist_key, map_key, hpix_out=None):
```

Comment:
What is the purpose of this commented out section? It looks like a copy of the previous state of the code.

Reply:
Thanks for catching, I removed this.  It was the old version of the helper functions that were replaced in this PR.